profiles/target_profile.go

```diff
@@ -53,9 +53,17 @@ type TargetProfileConnection struct {
 	Sp TargetProfileConnectionSpanner
 }
 
+type DataprocConfig struct {
+	Hostname   string
+	Subnetwork string
+	Port       string
+	TargetDB   string
+}
+
 type TargetProfile struct {
 	Ty   TargetProfileType
 	Conn TargetProfileConnection
+	Dc   DataprocConfig
```

Comment:
`TargetProfile` captures the configuration related to the destination, which would be Spanner during Dataproc based migrations. This doesn't look like the correct place to include `Dataproc` specific configurations.

Reply:
Moved to `SourceProfile.SourceProfileConfig.ShardConfigurationDataproc` 
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/520/commits/d65e9ff652f5c9adf3145b8c9c860c02076a8ad6